packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -468,16 +468,23 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin {
     ..._nonModifierKeys,
   };
 
+  /// Handling key event. It's public member for test code in editable_test.dart
   // TODO(goderbauer): doesn't handle extended grapheme clusters with more than one Unicode scalar value (https://github.com/flutter/flutter/issues/13404).
   // This is because some of this code depends upon counting the length of the
   // string using Unicode scalar values, rather than using the number of
   // extended grapheme clusters (a.k.a. "characters" in the end user's mind).
-  void _handleKeyEvent(RawKeyEvent keyEvent) {
+  void handleKeyEvent(RawKeyEvent keyEvent) {
     if (keyEvent is! RawKeyDownEvent || onSelectionChanged == null)
       return;
     final Set<LogicalKeyboardKey> keysPressed = LogicalKeyboardKey.collapseSynonyms(RawKeyboard.instance.keysPressed);
     final LogicalKeyboardKey key = keyEvent.logicalKey;
 
+    final bool isWeb = keyEvent.data is RawKeyEventDataWeb;
+    if (isWeb) {
+      // On web platforms, The key should be ignored because the cursor has already moved
```

Comment:
This comment is unclear, is the key down only apply to arrow key?

Reply:
This code replaces flutter engine's pull-request #13699. so ignore all keyEvent from input control on web.
Arrow key(_handleMovement function) and delete key(_handleDelete function) should be ignored. 

but I didn't check the keys in _handleShortcuts(it handles ctrl+c, ctrl+v, ctrl+x and ctrl+a) yet. I am going to check those keys should be ignored or not. then renew pull-request. Thank you.

Reply:
Ctrl+c, ctrl+v keys are working well. so I modify the code to ignore movement keys and delete key only.
Thank you.